docs/source/workflow_inputs.rst

```diff
@@ -4309,6 +4311,11 @@ Each heat pump water heater is entered as a ``/HPXML/Building/BuildingDetails/Sy
   .. [#] NumberofBedroomsServed only required if IsSharedSystem is true.
          Tank losses will be apportioned to the dwelling unit using its number of bedrooms divided by the total number of bedrooms served by the water heating system per `ANSI/RESNET/ICC 301-2022 <https://codes.iccsafe.org/content/RESNET3012022P1>`_.
          Each dwelling unit w/zero bedrooms should be counted as 1 bedroom -- e.g., a value of 3 should be entered for a shared system serving 3 studio (zero bedroom) apartments.
+  .. [#] Use HPWHInstalledProperly to specify whether to apply the HPWH COP adjustment for confined space, refer to `RESNET HERS Addendum 77 <https://www.resnet.us/about/standards/minhers/draft-pds-01-minhers-addendum-77-integrated-heat-pump-water-heaters/>`_ for conditions where it applies.
+  .. [#] HPWHContainmentVolume only required if HPWHInstalledProperly is false.
+  .. [#] HPWHContainmentVolume is used to adjust HPWH COP per `RESNET HERS Addendum 77 <https://www.resnet.us/about/standards/minhers/draft-pds-01-minhers-addendum-77-integrated-heat-pump-water-heaters/>`_.
```

Comment:
It's now pointing to the draft addendum, will need to replace after it's officially released, better link to use? Or just explain the approach details here without reference?

Reply:
@jmaguire1 Please review the doc changes